File: Labs/lab07/count_word.pl
#!/usr/bin/perl -w

use strict;

my $wordCount = 0;
my $chosenWord = $ARGV[0];
my %file_words = ();


while (my $line = <STDIN>) {
	$line =~ s/[^a-zA-Z]/ /g;
	$line = lc $line;
	my @words = split / /, $line;
	foreach my $word (@words) {
		if ($word eq "$chosenWord") {
			$wordCount += 1;
		}
	}
}

print "$chosenWord occurred $wordCount times\n";

File: Labs/lab07/count_word.py
#!/usr/bin/python

import sys, re

wordCount = 0
pattern = sys.argv[1]

for lines in sys.stdin:
	lines = re.sub("[^a-zA-Z]+", " ", lines)
	lines = lines.lower()
	lines = re.sub("\'", " ", lines)
	words = lines.split(" ")
	
	for word in words:
		if (word == pattern):
			print (word)
			wordCount += 1

print ("%s occurred %d times"%(pattern,wordCount))